.azure-pipelines/weekly-examples-update.yml

```diff
@@ -35,7 +32,10 @@ resources:
 extends:
   template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
   parameters:
-    pool: $(BuildAgent)
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
```

Comment:
Is this change expected?

Reply:
Yes, it's expected. The previous pool (1es-windows-ps-compute) has been restricted for AutoREST code generation functions.